assets/js/hooks/emoji.js

```diff
@@ -0,0 +1,27 @@
+import "../../vendor/emojimart";
+
+export const EmojiPicker = {
+    mounted() {
+        this.el.addEventListener("click", (event) => 
+            {
+                if(this.el.parentElement.querySelector("em-emoji-picker") == null) {
+                    const picker = new EmojiMart.Picker({onEmojiSelect: (data) => {updateEmojiInput(this.el.id, data.id); }, theme: "light", set: "native", autoFocus: true, onClickOutside: () => {picker.remove();}});
```

Comment:
```suggestion
                    const picker = new EmojiMart.Picker({onEmojiSelect: (data) => {updateEmojiInput(this.el.id, data.id); }, theme: "light", set: "native", autoFocus: true, onClickOutside: () => {picker.remove()}});
```
I'm not sure this works. We could search for a JS formatter or something - let's open some issues for these tiny things I'm pointing (as they need us to search a little bit before) 👀